### PR TITLE
OCPQE-12422: Fix check for IPsec enabled

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by jhou1** [comment]
/.assign @anuragthehatter @zhaozhanqi

**Comment by zhaozhanqi** [comment]
@liangxia could you add some information about the context for this changes.

File: lib/openshift/network.rb [hunk 1]
```diff
@@ -3,18 +3,13 @@
...
-    # resource_without_dots = "networksoperatoropenshiftio"
-    # lib/openshift/config_imageregistry_operator_openshift_io.rb` - for objects without unique name and you need full qualified name
+  class Network < ClusterResource
     RESOURCE = "network.operator"
 
     def network_type(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet)
-      rr.dig('spec', 'defaultNetwork', 'type')
+      return rr.dig('spec', 'defaultNetwork', 'type') || \
+             rr.dig('spec', 'networkType')
```
Comment (zhaozhanqi) [inline lib/openshift/network.rb:12]: network.operator do not have `networkType`

Comment (liangxia) [reply lib/openshift/network.rb]: Misread `network.operator.openshift.io` with `network.config.openshift.io`, update the PR accordingly.

**Pull request merged**